lib/faktory.rb

```diff
@@ -86,6 +86,10 @@ def self.faktory=(hash)
     @pool = Faktory::Connection.create(hash)
   end
 
+  def self.reset_pool
```

Comment:
Uh uh.  This is not a general purpose API that should be exposed.

Reply:
@mperham Do you have a recommendation for how you'd like to see this one resolved? I was running into problems where tests in this project would randomly fail based on run order and all I could figure what that it had to do with the pool being handled inconsistently between test mode and non-test mode. So this was what I ended up with that I could get to work consistently. Should this be a private method with a helper in the `Faktory::Testing` module that can `send` to it?  Or maybe do the `__reset_pool` thing so it looks less friendly?

Reply:
Or maybe I have the wrong idea about how/why/when to do something like this...

Reply:
A difficulty with testing should not "infect" the public API.  `Faktory.instance_variable_set(:@pool, nil)` is all you need in the test.

Reply:
Ok, cool. Thanks!